modules/core/src/de/diedavids/cuba/dataimport/binding/DatatypeFactory.groovy

```diff
@@ -75,15 +76,23 @@ class DatatypeFactory {
     }
 
     @SuppressWarnings('SimpleDateFormatMissingLocale')
-    private Date getDateValue(ImportConfiguration importConfiguration, String rawValue) {
-        SimpleDateFormat formatter = new SimpleDateFormat(importConfiguration.dateFormat)
-        formatter.parse(rawValue)
+    private Date getDateValue(ImportConfiguration importConfiguration, String rawValue, DataRow dataRow) {
+        try {
+            SimpleDateFormat formatter = new SimpleDateFormat(importConfiguration.dateFormat)
+            formatter.parse(rawValue)
+        } catch (ParseException e) {
+            log.warn("Date could not be read: '$rawValue' in [$dataRow]. Will be ignored.")
+        }
     }
 
     @SuppressWarnings('SimpleDateFormatMissingLocale')
-    private LocalDate getLocalDateValue(ImportConfiguration importConfiguration, String rawValue) {
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(importConfiguration.dateFormat)
-        LocalDate.parse(rawValue, formatter)
+    private LocalDate getLocalDateValue(ImportConfiguration importConfiguration, String rawValue, DataRow dataRow) {
+        try {
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern(importConfiguration.dateFormat)
+            LocalDate.parse(rawValue, formatter)
+        } catch (java.time.format.DateTimeParseException e) {
+            log.warn("LocalDate could not be read: '$rawValue' in [$dataRow]. Will be ignored.")
```

Comment:
perhaps we can make the `importConfiguration.dateFormat` here part of the warning message. Something like this:
```suggestion
            log.warn("LocalDate could not be read: '$rawValue' in [$dataRow], because it does not match the configured date format: '${importConfiguration.dateFormat}'. Will be ignored.")
```